src/ontology/edits/oeo-social.omn

```diff
@@ -846,6 +846,54 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1334",
         OEO_00010312 some OEO_00140004
     
     
+Class: OEO_00010328
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "An OEP user is a user who interacts with the Open Energy Platform.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1339
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1383",
+        rdfs:label "OEP user"@en
+    
+    SubClassOf: 
+        OEO_00000431
+    
+    
+Class: OEO_00010329
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A qualitist is an OEP user who mainly interacts with qualitive information.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1339
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1383",
+        rdfs:label "qualitist"@en
+    
+    SubClassOf: 
+        OEO_00010328
+    
+    
+Class: OEO_00010330
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A qualitist is an OEP user who mainly interacts with quantitive information.",
```

Comment:
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000115> "A quantitist is an OEP user who mainly interacts with quantatitive information.",
```

Reply:
... with quant**ita**tive information

Reply:
Oops, that was to quick a review... Thanks!